openssl-dynamic/src/main/c/tcn.h

```diff
@@ -106,85 +104,37 @@ jstring         tcn_new_stringn(JNIEnv *, const char *, size_t);
 #define AJP_TO_JSTRING(V)   (*e)->NewStringUTF((e), (V))
 
 #define TCN_CHECK_NULL(V, M, R)                      \
-    TCN_BEGIN_MACRO                                  \
+    NETTY_JNI_UTIL_BEGIN_MACRO                       \
         if (V == NULL) {                             \
             tcn_ThrowNullPointerException(e, #M);    \
             return R;                                \
         }                                            \
-    TCN_END_MACRO
+    NETTY_JNI_UTIL_END_MACRO
 
 #define TCN_FREE_JSTRING(V)      \
```

Comment:
is this used anywhere? if not can we remove it?

Reply:
Will check as a followup